tests/test_porcelain.py

```diff
@@ -23,14 +24,15 @@ def test_list_nodue(tmpdir, runner, create):
         'test.ics',
         'SUMMARY:Do stuff\n'
         'PERCENT-COMPLETE:12\n'
-        'PRIORITY:1\n'
+        'PRIORITY:4\n'
     )
 
     result = runner.invoke(cli, ['--porcelain', 'list'])
+    assert not result.exception
     assert (
-        result.output.strip() ==
```

Comment:
I'm not sure if you intentionally removed the `strip()`, but removing it makes the tests fail.